app/src/main/java/com/x8bit/bitwarden/data/platform/manager/di/PlatformManagerModule.kt

```diff
@@ -75,6 +77,14 @@ object PlatformManagerModule {
     fun provideAppForegroundManager(): AppForegroundManager =
         AppForegroundManagerImpl()
 
+    @Provides
+    @Singleton
+    fun provideBridgeServiceManager(
```

Comment:
One more rename

Reply:
4b91246e61dc934ab36b2fdfa37101349f392bc2 thank you :)